### PR TITLE
product page and cart

### PR DESCRIPTION
**Pull request by anasm11**

### CODE REVIEW

File: product-page.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.sidebar::-webkit-scrollbar {
+    display: none;
+  }
+
+  main{
+      display:grid;
+      gap:1rem;
+      grid-template-columns: repeat(auto-fit,250px);
```
Comment (Mohitk391) [inline product-page.css:39]: Suggestion: You can rem instead of pixels for responsive code

File: product-page.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      grid-column:1/-1;
+      display:flex;
+      justify-content: center;
+        align-items:center;
+        gap:1rem;
+  }
+
+  .input-with-icon{
+      display:flex;
+      background-color: white;
```
Comment (Mohitk391) [inline product-page.css:53]: Suggestion: you should use css variables instead of color names

File: product-page.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+a{
+    text-decoration:none;
+  }
+  
+  a:visited{
+    text-decoration:none;
+  }
+
+  .card{
+    width:250px;
```
Comment (Mohitk391) [inline product-page.css:84]: use rem

File: product-page.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    
+}
+.card img{
+  height:250px;
```
Comment (Mohitk391) [inline product-page.css:88]: use rem

**Review by Mohitk391 — APPROVED** [review]
Code looks clean and ui looks neat and clean as well...try to implement the suggesstions...otherwise its good to merge...

File: product-page.css (hunk 1 above)
Comment (anasm11) [reply product-page.css:39]: ok.. thank you for review

File: product-page.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+      border-radius:1rem;
+      border:1px solid;
+      overflow: hidden;
+      padding:0.3rem;
+  }
+
+  .input-with-icon>input{
+    border: none;
+    width:15rem;
+    padding:0.3rem;
```
Comment (abhisheksingh98) [inline product-page.css]: You can use px for less than 10px instead of rem

**Review by JuvitaSaini09 — APPROVED** [review]
Looks good to merge

File: product-page.css (hunk 5 above)
Comment (anasm11) [reply product-page.css]: ok ..thank you for review.

**Pull request merged**